pkg/commands/git_commands/patch.go

```diff
@@ -107,9 +108,11 @@ func (self *PatchCommands) MovePatchToSelectedCommit(commits []*models.Commit, s
 	err := self.rebase.PrepareInteractiveRebaseCommand(PrepareInteractiveRebaseCommandOpts{
 		baseShaOrRoot:  commits[baseIndex].Sha,
 		overrideEditor: true,
-		changeTodoActions: []ChangeTodoAction{
-			{sha: commits[sourceCommitIdx].Sha, newAction: todo.Edit},
-			{sha: commits[destinationCommitIdx].Sha, newAction: todo.Edit},
+		instruction: ChangeTodoActionsInstruction{
+			actions: []daemon.ChangeTodoAction{
```

Comment:
It's a little awkward that `daemon` now appears in client code like this, but I had to move the struct to `daemon.go` for the json unmarshalling. Any better ideas how to do this? I suppose I could move the struct to `utils/RebaseTodo.go` instead, not sure that's any better though.